src/main/java/org/openhab/binding/zwave/internal/protocol/ZWaveDeviceClass.java

```diff
@@ -379,6 +379,7 @@ public String getLabel() {
         SET_TOP_BOX_CONTROLLER(4, Generic.STATIC_CONTROLLER, "Set-Top Box Controller"),
         SUB_SYSTEM_CONTROLLER(4, Generic.STATIC_CONTROLLER, "Sub-System Controller"),
         TV_CONTROLLER(6, Generic.STATIC_CONTROLLER, "TV Controller"),
+        GATEWAY_CONTROLLER(7, Generic.STATIC_CONTROLLER, "Gateway Controller"),

```

Comment:
Done.  (I swear I saw the name "Gateway Controller" somewhere in the Z-Wave specs, but I can't find it now.)